### ISSUE TITLE
Support blocking all third-party requests from the current site in the pop-up window

### ISSUE THREAD

**Issue by ivysrono** [EFForg/privacybadger#2869]
Some sites randomly insert a large number of third-party requests, mainly ads but also many cross-site trackers.

These third-party requests are tracked to the user at the same time, and the current visit gets a completely different request than the next visit.

I hope to provide a switch for users to block all third-party requests from the current site with one click, and of course, you can also unblock them with one click. Somewhat similar to adblock extensions that enables/pauses filtering on the current page.

**Comment by ghostwords** [comment]
Hello and thanks for the suggestion!

Could you provide some examples of sites where it would be desirable to always block all third-party requests? Feel free to keep adding examples as you find them.

**Comment by ghostwords** [comment]
I recommend uBlock Origin's dynamic filtering: https://github.com/gorhill/uBlock/wiki/Dynamic-filtering:-quick-guide

Privacy Badger is meant to be a (mostly) install-and-forget privacy tool. Advanced request controls are out of scope for Privacy Badger, at least at this point.
